pyproject.toml

```diff
@@ -100,6 +100,7 @@ weaviate = ["weaviate-client"]
 pydantic = ["pydantic"]
 mssql = ["pyodbc"]
 qdrant = ["qdrant-client"]
+notebook = ["ipython", "ipywidgets", "jupyter"]
```

Comment:
please do not create extra for notebook. people will install that themselves. what we need is a poetry group (optional one) that will be used in order to run tests. so please revert changes to pyproject, add optional poetry group `notebook` instead